### DISCUSSION TITLE
How to sort groups at the My-Devices-list?

### DISCUSSION THREAD

**Discussion by n1smithy** [Ylianst/MeshCentral#6157] [Q&A] [answered]
Hello, everybody,

I want to sort groups by Name on the device-list. But I cannot find appropriate way to do so.

Can you help me?

Thanks and greetings!

**Comment by si458** [comment, marked as answer]
it already defaults to sort by group names, you just change the drop down
![image](https://github.com/Ylianst/MeshCentral/assets/765314/e6c4479b-7b5d-4ac9-af12-153f03a03b41)

**Reply by n1smithy** [reply]
[EDIT]
Ah, my fault! What I first answered only applies to empty groups. After a group was filled by a host it will be sorted by name.

Hm, It would be nice if one could sort groups manually or just to fix one or more groups at the top of the page (for instant-connections/-services for example).

[Old post]
But this does not sort by alphabet. It sorts chronologically.

**Reply by si458** [reply]
open a feature request for your `sort manually` as that would require like a drag/drop feature which is quite time consuming

but also then open another feature request for pin a `certain group` to the top your sort as that shouldnt be too hard i think as the sort it done client side, so we can store what group you want in session for when u later revisit the site :)
